CMakeLists.txt

```diff
@@ -189,7 +188,6 @@ set(SOURCES
         tools/sst_dump_tool.cc
         tools/dump/db_dump_tool.cc
         util/arena.cc
-        util/auto_roll_logger.cc
```

Comment:
Master branch moved some files. This one was not removed from list. File
already does no exist.
Dne 27. 1. 2016 19:36 napsal uživatel "Dmitri Smirnov" <
notifications@github.com>:

> In CMakeLists.txt
> <https://github.com/facebook/rocksdb/pull/959#discussion_r51027852>:
>
> > @@ -189,7 +188,6 @@ set(SOURCES
> >          tools/sst_dump_tool.cc
> >          tools/dump/db_dump_tool.cc
> >          util/arena.cc
> > -        util/auto_roll_logger.cc
>
> Why excluding this file is necessary? Otherwise, looks good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/rocksdb/pull/959/files#r51027852>.
>